### ISSUE TITLE
PyPI-Assistant extension for VSCode does not read pyproject.toml when using uv as project manager, but works with requirements.txt

### ISSUE THREAD

**Issue by mrchoke** [Twixes/pypi-assistant#31]
When working with a project that uses uv as the project manager, the PyPI-Assistant extension for VSCode does not recognize or read the pyproject.toml file. However, it continues to function properly when the project uses a requirements.txt file for dependency management.

The project structure is as follows:

```
.venv/
app/
.python-version
pyproject.toml
uv.lock
```

The contents of the pyproject.toml are:

```toml
[project]
name = "dev-reserve-api"
version = "0.1.0"
description = "Dev Reserve API"
readme = "README.md"
requires-python = ">=3.12"
dependencies = [
    "alembic>=1.13.2",
    "asyncpg>=0.29.0",
    "fastapi-users[sqlalchemy]>=13.0.0",
    "fastapi[all]>=0.115.0",
    "pandas>=2.2.2",
    "sqlalchemy[asyncio]>=2.0.34",
]
```

Expected Behavior:
The PyPI-Assistant extension should detect and properly read the pyproject.toml file for dependency management when uv is used as the project manager.

Actual Behavior:
The extension does not recognize the dependencies listed in pyproject.toml, but it works as expected with requirements.txt if used.

Steps to Reproduce:

1.	Create a project with the above structure, using uv as the project manager.
2.	Define the dependencies in the pyproject.toml file.
3.	Open the project in VSCode with the PyPI-Assistant extension installed.
4.	Notice that the extension does not recognize the dependencies listed in pyproject.toml.

Environment:

•	VSCode version: 1.93.1
•	PyPI-Assistant extension version: 2.1.2
•	Python version: 3.12
•	uv project manager version: 0.4.10
•	OS: Docker Container Debian 12

Let me know if additional information is needed! Thank you for your support.

**Comment by Twixes** [comment]
Hi @mrchoke! This is quite surprising, because I'm unable to reproduce the problem. When creating a `pyproject.toml` file with the contents you provided, I do see dependencies recognized:

<img width="408" alt="Screenshot 2024-10-06 at 19 24 39" src="https://github.com/user-attachments/assets/0cff0963-ba84-48f9-8de7-82a4dec0963a">

Is the problem still present on your end when you give it a try?

**Comment by mrchoke** [comment]
Apologies for the late reply. I have done further testing and can consistently reproduce the issue as follows:

This issue does not occur with a standard project on a local machine, but it arises when working within a devcontainer or over SSH remote. As shown in the attached image, the extension works correctly with requirements.txt, but fails to parse the pyproject.toml.

@Twixes 


![Screenshot 2024-10-07 at 12 57 55](https://github.com/user-attachments/assets/2e72d7ce-e01d-4c67-9d79-6e5231cf80bc)

Devcontainer

![Screenshot 2024-10-07 at 13 04 00](https://github.com/user-attachments/assets/7d3bec5b-635c-424d-a2a8-2e7709c099dd)

Remote SSH

Could you kindly look into this? Thank you!

**Comment by Twixes** [comment]
I see, thanks! I was able to debug this by looking at devcontainers specifically, and the problem should be solved in PyPI Assistant 2.1.9 now ([see diff](https://github.com/Twixes/pypi-assistant/compare/v2.1.2...v2.1.9)). Let me know if things are working well now, or something still missing! @mrchoke

**Comment by mrchoke** [comment]
Thank you for looking into and addressing the issue. However, after attempting to update the extension to version 2.1.9, it still didn’t work. Upon further investigation, I discovered that the actual problem might be due to VSCode not automatically recognizing the .toml file. Once I manually set the file type to TOML, your extension worked, even when reverting to an older version like 2.1.2. @Twixes 

![Screenshot 2024-10-28 at 08 28 08](https://github.com/user-attachments/assets/c777aabb-4a63-4aeb-95ae-fb8317aed36c)
![Screenshot 2024-10-28 at 08 29 20](https://github.com/user-attachments/assets/079c3c28-2b83-4a6e-a886-eb5f88f45b5f)
